JitsiConference.js

```diff
@@ -2530,6 +2553,57 @@ JitsiConference.prototype._onIceConnectionEstablished = function(
 
 };
 
+/**
+ * Called when the chat room reads a new list of properties from jicofo's
+ * presence. The properties may have changed, but they don't have to.
+ *
+ * @param {Object} properties - The properties keyed by the property name
+ * ('key').
+ * @private
+ */
+JitsiConference.prototype._propertiesChanged = function(properties) {
```

Comment:
What do you think of a name like updateProperties? It's more specific to what this does, but that might also be a problem depending on your intention.